indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1634,10 +1634,20 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
                                                registration.last_name.encode('utf-8')), text_format))
             data = registration.data_by_field
             for item in self._display:
-                if item.input_type == 'accommodation' and item.id in data:
-                    lp.append(Paragraph(data[item.id].friendly_data.get('choice').encode('utf-8'), text_format))
-                    lp.append(Paragraph(format_date(data[item.id].friendly_data.get('arrival_date')), text_format))
-                    lp.append(Paragraph(format_date(data[item.id].friendly_data.get('departure_date')), text_format))
+                if item.input_type == 'accommodation':
+                    if item.id in data:
+                        lp.append(Paragraph(data[item.id].friendly_data.get('choice').encode('utf-8'), text_format))
+                        arrival = data[item.id].friendly_data.get('arrival_date')
```

Comment:
why not use `arrival_date` here, too, and then overwrite the variable in the next line?